### DISCUSSION TITLE
Can't recieve email Verification on github (serious issue, locked out of my account)

### DISCUSSION THREAD

**Discussion by mrrezaee1995** [community/community#116513] [Other Feature Feedback, Questions, & Ideas]
Product Feedback

### Body

These is two days I can't sign in to my github account!
I enter the password, and it goes on page that says a verification email has sent but my inbox (gmail) was empty!
I tried forgot password button but no luck
virtually I'm am locked out of my account and I don't  know why
This is serious issue 
I have to create a new account with a different email just be able topost this issue 
Please help

**Comment by row77** [comment]
> 
> 
> Product Feedback
> 
> 
> 
> ### Body
> 
> 
> 
> These is two days I can't sign in to my github account!
> 
> I enter the password, and it goes on page that says a verification email has sent but my inbox (gmail) was empty!
> 
> I tried forgot password button but no luck
> 
> virtually I'm am locked out of my account and I don't  know why
> 
> This is serious issue 
> 
> I have to create a new account with a different email just be able topost this issue 
> 
> Please help
> 
>
